src/mudlet.cpp

```diff
@@ -156,7 +156,7 @@ mudlet::mudlet()
 , replayToolBar( 0 )
 , moduleTable( 0 )
 , mStatusBarState( statusBarAlwaysShown )
-, mIsToDisplayMapAuditErrorsToConsole( false )
+, mshowMapAuditErrors( false )
```

Comment:
Called me picky but I like to name `bool`eans with something beginning with "is"... 😐